### PR TITLE
SQL implementation

### PR DESCRIPTION
**Pull request by grahamegrieve**

### CODE REVIEW

File: org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/renderers/DBBuilder.java [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+        "PRIMARY KEY (Key))\r\n");
+  }
+
+
+  public String processSQL(String sql) {
+    if (con == null) {
+      return "<span style=\"color: maroon\">Error processing SQL: SQL is not set up properly</span>";
+    }
+
+    sql = sql.trim();
```
Comment (dotasek) [inline org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/renderers/DBBuilder.java:413]: Null sql here could blow up. I'd suggest doing something like this in the catch block:

```java
 if (sql == null) {
        throw new IllegalArgumentException("Param sql cannot be null.");
      }
     
      sql = sql.trim();
```

**Review by dotasek — CHANGES_REQUESTED** [review]

File: org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/renderers/DBBuilder.java (hunk 1 above)
Comment (grahamegrieve) [reply org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/renderers/DBBuilder.java:413]: ok.

Comment (dotasek) [reply org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/renderers/DBBuilder.java]: I moved that into the try block. Feel free to revert that commit if it's improper, but I think it's safer to do it that way.

Comment (grahamegrieve) [reply org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/renderers/DBBuilder.java]: no that's ok

**Pull request merged**